src/components/Summary/AddNewScoreRedux/index.js

```diff
@@ -0,0 +1,268 @@
+            type="text"
+            placeholder="please leave your name"
+            style={{ width: '240px', height: '50px', marginTop: '-10px' }}
+            onChange={this.onDetectNullValue}
+          />
+        </Row>
+        <form>
+          <Row type="flex" align="middle">
+            <Col type="flex">
+              <Row type="flex">
+                <h4 style={{ display: 'inline', marginTop: '120px' }}>
+                  Skills
+                </h4>
+                <Field
+                  style={{ marginLeft: '145px', marginTop: '110px' }}
+                  name="quality"
+                  component={RateScore}
+                  onChange={this.onDetectNullValue}
+                />
+              </Row>
+              <Row type="flex">
+                <h4 style={{ display: 'inline', marginTop: '10px' }}>
+                  Potential
+                </h4>
+                <Field
+                  style={{ marginLeft: '120px', marginBottom: '-130px' }}
+                  name="completeness"
+                  component={RateScore}
+                  onChange={this.onDetectNullValue}
+                />
+              </Row>
+              <Row type="flex">
+                <h4 style={{ display: 'inline', marginTop: '10px' }}>
+                  Adaptability
+                </h4>
+                <Field
+                  name="hint"
+                  style={{ marginLeft: '100px' }}
+                  component={RateScore}
+                  onChange={this.onDetectNullValue}
+                />
+              </Row>
+            </Col>
+            <>
+              <Col
+                type="flex"
+                span={10}
+                offset={5}
+                style={{ marginTop: '-60px' }}
+              >
+                <h2 style={{ marginBottom: '150px' }}>Comment </h2>
+
+                <Field
+                  name="comment"
+                  component={CommentScore}
+                  type="text"
+                  placeholder="please leave your comment"
+                  style={{ marginTop: '500px' }}
+                />
+              </Col>
+              <Row>
+                <Button
+                  align="middle"
+                  style={{
+                    marginTop: '50px',
+                    marginLeft: '150px',
+                    width: '100px',
+                  }}
+                  disabled={!shouldEnableSubmitButton}
+                  htmlType="submit"
+                  onClick={this.handleSubmit}
+                  onKeyPress={e => {
+                    if (e.key === 'Enter') e.preventDefault();
+                  }}
+                >
+                  Add Score
+                </Button>
+              </Row>
+            </>
+          </Row>
+        </form>
+      </>
+    );
+  }
+}
+
+//const testreduxstate=connect(mapStateToProps, null)(AddNewScoreFormRedux);
+function mapStateToProps(state) {
+  return {
+    formStates: getFormValues('AddScore')(state),
```

Comment:
`AddScore` is a form name. You could put it in a constant for better maintenance.

``` javascript
const formName = 'AddScore';
```